test/cmd/newapp.sh

```diff
@@ -612,5 +610,23 @@ os::cmd::expect_success 'oc new-app php-55-centos7:latest https://github.com/ope
 os::cmd::expect_success_and_text 'oc get bc test8 --template={{.spec.strategy.sourceStrategy.from.name}}' 'php-55-centos7:latest'
 os::cmd::expect_success 'oc delete project new-app-syntax'
 
+# new-app docker build strategy with binary input
+os::cmd::expect_success 'oc project ${default_project}'
+os::cmd::expect_success 'oc delete all,templates --all'
+os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
+os::cmd::try_until_success 'oc get imagestreamtags ruby:2.0' # need to wait until tags are available!?
+os::cmd::expect_failure_and_text 'oc new-app --strategy=docker --name my-docker-app' 'none of the arguments provided could be classified as a source code location'
+os::cmd::expect_success_and_text 'oc new-app --strategy=docker --binary --name my-docker-app' 'A binary build was created'
+os::cmd::expect_success_and_text 'oc start-build my-docker-app --from-dir=./test/testdata/testapp' 'build.build.openshift.io/my-docker-app-1 started'
```

Comment:
you don't need to run the build.

Reply:
OK. My intention was to show no build was automatically started when creating the app. But I can remove it.

On another note, I was tempted to split all start-build commands into its own startbuild.sh test suite. They are really two different commands and this test suite is already so big and takes a long time to run. WDYT?